references/recaptcha-enterprise/data-proxy-v1/apiproxy/proxies/default.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2022 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<ProxyEndpoint name="default">
+    <PreFlow name="PreFlow"/>
+    <Flows>
+        <Flow name="GET /headers">
+            <Condition>(proxy.pathsuffix MatchesPath "/headers") and 
+            (request.verb = "GET")</Condition>
+            <Request>
+                <Step>
+                    <Name>FC-RecaptchaEnterprise</Name>
+                </Step>
+            </Request>
+            <Response/>
+        </Flow>
+        <Flow name="404 - Not Found">
+            <Request>
+                <Step>
+                    <Name>RF-404NotFound</Name>
+                </Step>
+            </Request>
+            <Response/>
+        </Flow>
+    </Flows>
+    <PostFlow name="PostFlow"/>
+    <HTTPProxyConnection>
+        <BasePath>/data/v1</BasePath>
```

Comment:
can we make this base path mention recaptcha. There might be other DevRel resources conflicting with this base path and it's not obvious that this is releated to the recaptcha reference.